skills/axon/examples/basic-scrape.sh

```
@@ -0,0 +1,192 @@
+# Purpose: Demonstrates simple web page scraping with Axon
+# Use Case: Extracting documentation or blog content for analysis
+
+set -euo pipefail
+
+# ============================================================================
+# CONFIGURATION
+# ============================================================================
+
+# Load credentials from .env file
+if [[ -f ~/claude-homelab/.env ]]; then
+    source ~/claude-homelab/.env
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)